replicate/client.py

```diff
@@ -348,6 +348,23 @@ def close(self) -> None:
         self._wrapped_transport.close()  # type: ignore
 
 
+def _get_api_token_from_environment() -> Optional[str]:
+    """Get API token from cog current scope if available, otherwise from environment."""
+    try:
+        import cog
+
+        if hasattr(cog, "current_scope"):
+            scope = cog.current_scope()
+            if scope and hasattr(scope, "context") and isinstance(scope.context, dict):
+                for key, value in scope.context.items():
+                    if key.upper() == "REPLICATE_API_TOKEN":
+                        return scope.context[key]
```

Comment:
Weak opinion:

```suggestion
        for key, value in cog.current_scope().context.items():
            if key.upper() == "REPLICATE_API_TOKEN":
                return scope.context[key]
```

We're catching all exceptions so we don't need to be defensive about checking for existence and typing.

Reply:
this will also fix the lint

Reply:
Hmm, that's a good point.